common/src/commonMain/kotlin/org/hisp/dhis/common/screens/CheckboxScreen.kt

```diff
@@ -0,0 +1,59 @@
+package org.hisp.dhis.common.screens
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import org.hisp.dhis.common.screens.previews.CheckboxPreview
+import org.hisp.dhis.common.screens.previews.TextCheckboxPreview
+import org.hisp.dhis.mobile.ui.designsystem.component.ColumnComponentContainer
+
+@Composable
+fun CheckboxScreen() {
+    val option1 = "Option 1"
+    val option2 = "Option 2"
+    val option3 = "Option 3"
+    val option4 = "Option 4"
+
+    val state1 = mutableStateOf(true)
+    val state2 = mutableStateOf(false)
+    val state3 = mutableStateOf(true)
+    val state4 = mutableStateOf(false)
+
+    Column(modifier = Modifier.padding(10.dp)) {
```

Comment:
Here there are many nested columns, maybe you could just use ColumnComponentContainer and add Text for components description/title